### PR TITLE
Update fileio.rst

### PR DESCRIPTION
**Pull request by srpyprime**

### CODE REVIEW

File: doc/source/user_guide/fileio.rst [hunk 1]
```diff
@@ -82,22 +82,33 @@ Four CAD import routes are available in PyPrimeMesh:
 
  * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.
 
-.. note::
-    Program Controlled and SpaceClaim route imports JT files containing only faceted data. Workbench and
-    Native does not support importing JT files containing only faceted data.
+ * ``Discovery``: Uses Discovery to import supported ``*.dsco`` and  ``*.scdocx`` files.
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:85]:
```suggestion
 * ``Discovery``: Uses Discovery to import supported DSCO and SCDOCX files.
```
Style guide says to use all capital letters for file extensions when used standalone.

File: doc/source/user_guide/fileio.rst [hunk 2]
```diff
@@ -82,22 +82,33 @@ Four CAD import routes are available in PyPrimeMesh:
...
 
 To view the CAD supported for the Workbench route on different platforms, see
 `CAD Support <https://www.ansys.com/it-solutions/platform-support>`_ on the Ansys website. 
 
 .. note::
-    When deploying scripts using SpaceClaim or Workbench CAD readers, ensure that the CAD configuration
+*   Program Controlled and SpaceClaim route imports JT files containing only faceted data. Workbench and
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:91]: Having trouble understanding the initial capital letters in the first few words here and in the next few lines. Only capitalize proper nouns. Any UI elements in a product should be in bold font.

File: doc/source/user_guide/fileio.rst [hunk 3]
```diff
@@ -82,22 +82,33 @@ Four CAD import routes are available in PyPrimeMesh:
...
+    Native does not support importing JT files containing only faceted data.
+
+*   Transmogrifier supports files from Workbench(pmdb,agdb), JT(faceted data from ``(*.jt/*.plmxml)``,
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:94]:
```suggestion
*   Transmogrifier supports Workbench PMDB and AGDB files, JT faceted data JT and PLMXML files, 
```
I'm not sure about that second part, but that partly demonstrates how the current format isn't all that clear.

File: doc/source/user_guide/fileio.rst [hunk 4]
```diff
@@ -82,22 +82,33 @@ Four CAD import routes are available in PyPrimeMesh:
...
+	XC ``(.fmdb, .sab/.sat, .stl, .msh/.tgf, .x_t/.x_b)``.
+
+*  SpaceCliam supports files with ``*.scdoc``, ``*.scdocx`` extension.
+
+*  Discovery supports files with ``*.dsco``, ``*.scdocx`` extension.
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:99]:
```suggestion
*  SpaceCliam supports files with SCDOC and SCDOCX extensions.

*  Discovery supports files with DSCO and SCDOCX extensions.
```

File: doc/source/user_guide/fileio.rst [hunk 5]
```diff
@@ -70,32 +70,43 @@ Alternatively, you can use the :class:`Mesh <ansys.meshing.prime.lucid.Mesh>` cl
...
-   and the ``Workbench`` route for all other file formats.  
+ * ``Program controlled``: chooses the CAD reader route based on the extension of the provided CAD file, as follows;
+   native for FMD, SAT, SAB, X_T, X_B, JT, PLMXML, and STL;
+   SpaceClaim for SCDOC and SCDOCX;
+   Discovery for DSCO;
+   Workbench for all other extensions.
 
- * ``Native``: Imports selected natively supported file formats like FMD ``(*.fmd)``, ACIS ``(*.sat, *.sab)``,
-   Parasolid ``(*.x_t, *.x_b)``, JTOpen ``(*.jt, *.plmxml)``, and STL ``(*.stl)``. 
+ * ``Native``: natively supported file format extensions are FMD, ACIS (SAT and SAB),
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:79]:
```suggestion
 * ``Native``: Natively supported file format extensions are FMD, ACIS (SAT and SAB),
```

File: doc/source/user_guide/fileio.rst [hunk 6]
```diff
@@ -70,32 +70,43 @@ Alternatively, you can use the :class:`Mesh <ansys.meshing.prime.lucid.Mesh>` cl
...
+   Parasolid (X_T and X_B), JTOpen (JT and PLMXML), and STL.
 
- * ``SpaceClaim``: Uses SCDM to import supported CAD files from the SpaceClaim reader. Only the
-   Windows platform supports importing SpaceClaim files.  
+ * ``SpaceClaim``: uses SpaceClaim to import supported CAD files from the SpaceClaim reader. Only the
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:82]:
```suggestion
 * ``SpaceClaim``: Uses SpaceClaim to import supported CAD files from the SpaceClaim reader. Only the
```

File: doc/source/user_guide/fileio.rst [hunk 7]
```diff
@@ -70,32 +70,43 @@ Alternatively, you can use the :class:`Mesh <ansys.meshing.prime.lucid.Mesh>` cl
...
+   Windows platform supports importing files using the SpaceClaim reader.
 
- * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.
+ * ``Discovery``: uses Discovery to import supported CAD files from the Discovery reader. Only the
+   Windows platform supports importing files using the Discovery reader.
+
+ * ``Workbench``: uses Workbench to import supported CAD files from the Workbench reader.
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:88]:
```suggestion
 * ``Discovery``: Uses Discovery to import supported CAD files from the Discovery reader. Only the
   Windows platform supports importing files using the Discovery reader.

 * ``Workbench``: Uses Workbench to import supported CAD files from the Workbench reader.
```

File: doc/source/user_guide/fileio.rst [hunk 8]
```diff
@@ -70,32 +70,43 @@ Alternatively, you can use the :class:`Mesh <ansys.meshing.prime.lucid.Mesh>` cl
...
 
-.. note::
-    Program Controlled and SpaceClaim route imports JT files containing only faceted data. Workbench and
-    Native does not support importing JT files containing only faceted data.
 
 To view the CAD supported for the Workbench route on different platforms, see
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:91]:
```suggestion
To view the CAD files supported for the Workbench route on different platforms, see
```

File: doc/source/user_guide/fileio.rst [hunk 9]
```diff
@@ -70,32 +70,43 @@ Alternatively, you can use the :class:`Mesh <ansys.meshing.prime.lucid.Mesh>` cl
...
+    * When deploying scripts using SpaceClaim, Discovery or Workbench CAD reader routes, ensure that the CAD configuration
+      and in-app defaults are consistent in the deployed environment.
+
+    * You must install and configure Workbench CAD readers or plug-ins (Ansys Geometry Interfaces)
+      while installing Ansys Workbench.
+
+    * In order to preserve shared topology the Workbench CAD reader route must be used.
+
+    * Patterned name selections are not supported for the SpaceClaim or Discovery CAD reader routes.
+      To import patterned named selections the Workbench CAD reader route can be used.
```
Comment (PipKat) [inline doc/source/user_guide/fileio.rst:107]:
```suggestion
      To import patterned named selections, the Workbench CAD reader route can be used.
```

**Review by PipKat — APPROVED** [review]

**Review by hlee0122 — APPROVED** [review]

File: doc/source/user_guide/fileio.rst (hunk 2 above)
Comment (waltersma) [reply doc/source/user_guide/fileio.rst]: @PipKat resolving as outdated. LGTM

**Pull request merged**